<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Fossil::Internal::CloneOptionsPanel</class>
 <widget class="QWidget" name="Fossil::Internal::CloneOptionsPanel">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>283</width>
    <height>166</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string notr="true"/>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QGroupBox" name="optionsGroupBox">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Minimum">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="title">
      <string>Options</string>
     </property>
     <property name="flat">
      <bool>true</bool>
     </property>
     <property name="checkable">
      <bool>true</bool>
     </property>
     <property name="checked">
      <bool>false</bool>
     </property>
     <layout class="QGridLayout" name="gridLayout">
      <item row="0" column="0">
       <widget class="QLabel" name="label">
        <property name="toolTip">
         <string>Privileged user added automatically to the created local repository.</string>
        </property>
        <property name="text">
         <string>Admin User:</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QLineEdit" name="adminUserLineEdit">
        <property name="toolTip">
         <string>Privileged user added automatically to the created local repository.</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="sslIdentityFileLabel">
        <property name="toolTip">
         <string>SSL/TLS client identity key to use if requested by the server.</string>
        </property>
        <property name="text">
         <string>SSL/TLS Identity:</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="Utils::PathChooser" name="sslIdentityFilePathChooser" native="true">
        <property name="toolTip">
         <string>SSL/TLS client identity key to use if requested by the server.</string>
        </property>
       </widget>
      </item>
      <item row="2" column="0" colspan="2">
       <widget class="QCheckBox" name="includePrivateCheckBox">
        <property name="toolTip">
         <string>Allow transfer of private branches.</string>
        </property>
        <property name="text">
         <string>Include private branches</string>
        </property>
       </widget>
      </item>
      <item row="3" column="0" colspan="2">
       <widget class="QCheckBox" name="disableAutosyncCheckBox">
        <property name="toolTip">
         <string>Disable automatic pull/push of local changes to the remote repository.</string>
        </property>
        <property name="text">
         <string>Disable auto-sync</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>4</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>Utils::PathChooser</class>
   <extends>QWidget</extends>
   <header location="global">utils/pathchooser.h</header>
   <container>1</container>
   <slots>
    <signal>editingFinished()</signal>
    <signal>browsingFinished()</signal>
   </slots>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
